readme.md

```diff
@@ -200,104 +107,224 @@ $ adb forward tcp:9222 localabstract:chrome_devtools_remote
 $ lighthouse --disable-device-emulation --disable-cpu-throttling https://mysite.com
 ```
 
-## Tests
+## Using programmatically
 
-Some basic unit tests forked are in `/test` and run via mocha. ESLint also checks for style violations.
+The example below shows how to setup and run Lighthouse programmatically as a Node module.
```

Comment:
> When looking into a lib, I do really like seeing install + different usages in the main readme.

Totally agree, I just also would find it off-putting if the usage example was so complicated :) I say let's leave it here, and I'll file issue to improve the API.